installers/kubectl/client-setup.sh

```diff
@@ -14,6 +14,40 @@
 # This script should be run after the operator has been deployed
 export PGO_OPERATOR_NAMESPACE="${PGO_OPERATOR_NAMESPACE:-pgo}"
 PGO_USER_ADMIN="${PGO_USER_ADMIN:-pgouser-admin}"
+PGO_CLIENT_VERSION="${PGO_CLIENT_VERSION:-v4.3.0}"
+PGO_CLIENT_URL="https://github.com/CrunchyData/postgres-operator/releases/download/${PGO_CLIENT_VERSION}"
+LINUX_BIN="pgo"
+DARWIN_BIN="pgo-mac"
+VALID_OS_OPTIONS="Linux, Darwin"
+
+# Checks operating system and determines which binary to download
+UNAME_RESULT=$(uname)
+if [ "${UNAME_RESULT}" == "Linux" ]
```

Comment:
For single brackets (test) use `=`. For bash pattern matching, use `[[` and `==`.

Reply:
Is this how it should be? I'm not sure i understand you comment
```
if [[ "${UNAME_RESULT}" == "Linux" ]]
then
    BIN_NAME="pgo"
elif [[ "${UNAME_RESULT}" == "Darwin" ]]
then
```